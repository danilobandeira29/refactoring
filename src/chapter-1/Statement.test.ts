import statement from "./Statement";
const invoices = require('./mocks/Invoices.json')
const plays = require('./mocks/Plays.json')

describe("Statement SUITE", function() {
    test('statement', function() {
        const statement_string = statement(invoices, plays);
        expect(statement_string).toMatchSnapshot();
    });
});
