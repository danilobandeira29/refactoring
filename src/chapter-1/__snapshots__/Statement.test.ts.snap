// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Statement SUITE statement 1`] = `
"<h1>Statement for BigCo</h1>
<table>
<tr><th>play</th><th>seats</th><th>cost</th></tr>
 <tr><td>Hamlet</td><td>55</td>
 <td>$650.00</td>
 <tr><td>As You Like It</td><td>35</td>
 <td>$580.00</td>
 <tr><td>Othello</td><td>40</td>
 <td>$500.00</td>
</table>
<p>Amount owed is <em>$1,730.00</em></p>
<p>You earned <em>47</em> credits</p>
"
`;
